src/OrchardCore.Modules/OrchardCore.Workflows/Migrations.cs

```diff
@@ -146,5 +146,37 @@ public int UpdateFrom2()
 
             return 3;
         }
+
+        // This code can be removed in a later version.
+        public int UpdateFrom3()
+        {
+            //For PostgreSQL
+            SchemaBuilder.AlterIndexTable<WorkflowBlockingActivitiesIndex>(table => table
+                .DropIndex("IDX_WorkflowBlockingActivitiesIndex_DocumentId_Activity"));
```

Comment:
@jptissot Can you confirm that it creates this index on your PostgreSQL database on the WorkflowBlockingActivities table.